sdmetrics/single_table/new_row_synthesis.py

```diff
@@ -0,0 +1,116 @@
+"""New Row Synthesis metric for single table."""
+import warnings
+
+import numpy as np
+import pandas as pd
+
+from sdmetrics.goal import Goal
+from sdmetrics.single_table.base import SingleTableMetric
+
+
+class NewRowSynthesis(SingleTableMetric):
+    """NewRowSynthesis Single Table metric.
+
+    This metric measures whether each row in the synthetic data is new,
+    or whether it exactly matches a row in the real data.
+
+    Attributes:
+        name (str):
+            Name to use when reports about this metric are printed.
+        goal (sdmetrics.goal.Goal):
+            The goal of this metric.
+        min_value (Union[float, tuple[float]]):
+            Minimum value or values that this metric can take.
+        max_value (Union[float, tuple[float]]):
+            Maximum value or values that this metric can take.
+    """
+
+    name = 'NewRowSynthesis'
+    goal = Goal.MAXIMIZE
+    min_value = 0
+    max_value = 1
+
+    @classmethod
+    def compute(cls, real_data, synthetic_data, metadata=None, numerical_match_tolerance=0.01,
+                synthetic_sample_size=None):
+        """Compute this metric.
+
+        This metric looks for matches between the real and synthetic data for
+        the compatible columns. This metric also looks for matches in missing values.
+
+        Args:
+            real_data (Union[numpy.ndarray, pandas.DataFrame]):
+                The values from the real dataset.
+            synthetic_data (Union[numpy.ndarray, pandas.DataFrame]):
+                The values from the synthetic dataset.
+            metadata (dict):
+                Table metadata dict.
+            numerical_match_tolerance (float):
+                A float >0.0 representing how close two numerical values have to be
```

Comment:
could we use `bigger than 0` instead of `>0.0` and `Defaults to `0.01` at the end?